bats/README.md

```diff
@@ -0,0 +1,143 @@
+## Overview
+
+BATS is a testing framework for Bash shell scripts that provides supporting libraries and helpers for customizable server-side test automation.
+
+## Setup
+
+It's important to have a Rancher Desktop CI build installed with privileged rights before running the BATS tests.
```

Comment:
Let's remove "with privileged rights" here, assuming it just means "installed for all users". I'll create a PR to fall back on a per-user installation if the system-wide location does not exist.

```suggestion
It's important to have a Rancher Desktop CI or release build installed before running the BATS tests.
```